kong.conf.default

```diff
@@ -1826,3 +1826,29 @@ G# -----------------------
 
 #wasm_filters_path =              # Path to the directory containing Wasm filters
                                   # that Kong must load on startup.
+
+#wasm_shm_kv =                    # Define a shared key/value memory zone accessible
+                                  # through all Contexts and by all nginx worker
+                                  # processes.
+
+#wasm_shm_queue =                 # Define a shared queue memory zone accessible
+                                  # through all nginx contexts and by all nginx
+                                  # worker processes.
```

Comment:
Do these things have a default?

Reply:
I think of "shm" as a memory zone - I can CRUD it.
When I read a 'queue', I think of it as a data-structure for queueing purposes and "memory zone" confuses me. It is added here because it is an implementation detail.
The most important user-facing detail is that the queue is shared across worker processes.

IF I understand this right, I recommend removing "memory zone" from this sentence.

Reply:
Agreed.